### ISSUE TITLE
"No connected database" when trying to send rageshakes

### ISSUE THREAD

**Issue by jryans** [vector-im/riot-web#13290]
Hm, well I don't get the Something went wrong! message on refresh anymore, but submitting logs does result in an "No connected database" still.

And that doesn't go away even after logging in and out... so going to assume something else is causing that. Is there any other way I can get debug logs to you guys ooi?

_Originally posted by @anoadragon453 in https://github.com/vector-im/riot-web/issues/13287#issuecomment-616606502_

**Comment by jryans** [comment]
@anoadragon453, are you using a private window in Firefox? Do you see this immediately after login without any further actions?

**Comment by anoadragon453** [comment]
@jryans Correct. Reproducible with a new account, on https://riot.im/develop

Possibly useful log lines:

![image](https://user-images.githubusercontent.com/1342360/79787149-74dbe300-833e-11ea-8110-a6b991d8e3c8.png)

Just prints "Sending bug report" when clicking "Send logs":

![image](https://user-images.githubusercontent.com/1342360/79787177-7efde180-833e-11ea-8dd7-5f4adde13ebb.png)

**Comment by t3chguy** [comment]
Well the logs are stored in IDB which is not available in Firefox InPrivate so it makes sense

**Comment by anoadragon453** [comment]
oh, well then

**Comment by jryans** [comment]
It would be nice to have some fallback path for recording and reporting rageshake data when IDB is not available, but it does appear like that could be complex to pull off, since the implementation is heavily IDB based right now.

**Comment by jryans** [comment]
@anoadragon453 As a workaround for now, you could use Firefox's containers feature to login with test accounts instead of private windows so that you'll have regular IDB access as expected.

**Comment by anoadragon453** [comment]
> @anoadragon453 As a workaround for now, you could use Firefox's containers feature to login with test accounts instead of private windows so that you'll have regular IDB access as expected.

Have been doing so and it works great, thanks!

**Comment by t3chguy** [comment]
Firefox fixed this https://www.mozilla.org/en-US/firefox/115.0/releasenotes/
